nixd/docs/user-guide.md

```diff
@@ -64,6 +64,17 @@ nix build -L .#
 
 ### Configuration
 
+`.nixd.json`, the configuration of `nixd`, supports [json schema](https://json-schema.org/).
+So if your editor supports
+[LSP](https://microsoft.github.io/language-server-protocol/implementors/servers/),
+you can get completions, diagnostics and more:
+
+![completion](https://github.com/nix-community/nixd/assets/32936898/013367c6-63d8-4bba-9057-c5701c2f36c1)
+
+![diagnostic](https://github.com/nix-community/nixd/assets/32936898/d285eb69-f023-4573-a8e5-8d5501ae3d16)
+
+The pictures are [neovim](https://neovim.org/) with the plugin
```

Comment:
Maybe just remove this line or use footnotes.

Reply:
Done.